Makefile

```diff
@@ -29,6 +34,21 @@ tools: $(TOOLS)
 $(TOOLS): oci-%:
 	go build -ldflags "-X main.gitCommit=${COMMIT}" ./cmd/$@
 
+.PHONY: man install uninstall
+
+_man:
+	go-md2man -in "cmd/oci-unpack/oci-unpack.1.md" -out "oci-unpack.1"
+	go-md2man -in "cmd/oci-image-validate/oci-image-validate.1.md" -out "oci-image-validate.1"
+	go-md2man -in	"cmd/oci-create-runtime-bundle/oci-create-runtime-bundle.1.md" -out "oci-create-runtime-bundle.1"
+
+install: _man
+	install -m 755 $(TOOLS) /usr/local/bin/
+	install -m 644 $(MAN) /usr/local/share/man/man1
+
+uninstall: clean
+	rm -f /usr/local/share/man/man1/oci-create-runtime-bundle.1 /usr/local/share/man/man1/oci-image-validate.1 /usr/local/share/man/man1/oci-unpack.1
+	rm -f /usr/local/bin/oci-create-runtime-bundle /usr/local/bin/oci-image-validate /usr/local/bin/oci-unpack
```

Comment:
rm -f $(patsubst %,/usr/local/share/man/man1/%,$(MAN)) $(patsubst %,/usr/local/bin/%,$(TOOLS))

although you probably want variables for `/usr/local/share/man` and `/usr/local/bin/` in case the user wants to override them.